README.org

```diff
@@ -759,6 +759,8 @@ For additional git related emacs packages to use or to get inspiration from, tak
      - [[https://www.emacswiki.org/emacs/CategoryGnus][Gnus]] - =[built-in]= Reading e-mail and Usenet news.
      - [[https://www.emacswiki.org/emacs/MessageMode][Messages]] - =[built-in]= Composing and sending e-mail inside Emacs.
      - [[http://www.djcbsoftware.nl/code/mu/mu4e.html][mu4e]] - An e-mail client for Emacs.
+       - [[https://github.com/iqbalansari/mu4e-alert/tree/91f0657c5b245a9de57aa38391221fb5d141d9bd][mu4e-alert ]]- Desktop Notification for mu4e.
```

Comment:
@innerout , modify the URL to: https://github.com/iqbalansari/mu4e-alert . 

Also, ideally, the description after the URL matches or slightly modifies the description provided on the repo: "Desktop notifications and modeline display for mu4e"